build-tools/packages/build-tools/src/typeValidator/testGeneration.ts

```diff
@@ -15,6 +15,7 @@ export function buildTestCase(
 	useType: TestCaseTypeData,
 	isCompatible: boolean,
 ) {
+	// Checks if test is compatible and old/current types exist
```

Comment:
I think this is accurate, right? Feels a bit clearer IMO.
```suggestion
	// Skip If the old/new types are incompatible because one of them doesn't exist (i.e new
	// type that didn't exist before, or type that was removed in the new version)
```